jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/impl/DynAttrFacetInfo.java

```diff
@@ -222,9 +222,12 @@ public Collection<String> getDynAttrViewTargetComponentIds(MetaClass metaClass,
 
     private void findTargetElementNames(List<String> targetElementList, Element searchElement) {
         for (var child : searchElement.elements()) {
-            if (child.getQName().getName().equals(JmixFormLayout.QUALIFIED_XML_NAME) ||
-                    child.getQName().getName().equals("dataGrid")) {
-                targetElementList.add(child.attributeValue("id"));
+            if (child.getQName().getName().equals(FORM_LAYOUT_QUALIFIED_XML_NAME) ||
```

Comment:
Again, what about my previous comment?

> It seems that constant name is not correct. Is this a qualified name? In my understanding a qualified name is the name that consist of a schema name and a local name, e.g. xsl:template. I don't see any schema here.

Even in the code you first get the **qualified** name by `child.getQName()` and then you get the **local** name from the QName object: `child.getQName().getName()`. So let's remove the word QUALIFIED from constant names. For example, DATAGRID_QUALIFIED_XML_NAME may be just DATAGRID_XML_TAG